lib/matplotlib/contour.py

```diff
@@ -1183,6 +1183,9 @@ def _contour_level_args(self, z, args):
         if self.filled and len(self.levels) < 2:
             raise ValueError("Filled contours require at least 2 levels.")
 
+        # Sort the contour levels.
+        if len(self.levels) > 1:
+            self.levels = list(sorted(set(self.levels)))
```

Comment:
(any dev should feel free to dismiss the review once they consider the issues addressed)